### PR TITLE
Anthony/add ssr support

### PR DESCRIPTION
**Pull request by anth-cole**
Needs some tidy up but here is the GPS support for the SSR RTCM messages @mfine @martin-swift

### CODE REVIEW

File: c/src/rtcm3_sbp.c [hunk 1]
```diff
@@ -246,6 +247,29 @@ void rtcm2sbp_decode_payload(const uint8_t *payload,
...
+    }
+    case 1060:
+    case 1066: {
+      rtcm_msg_orbit_clock msg_orbit_clock;
+      if (RC_OK == rtcm3_decode_orbit_clock(&payload[byte], &msg_orbit_clock)) {
+        rtcm3_ssr_orbit_clock_to_sbp(&msg_orbit_clock, state);
+      }
+      break;
+    }
+    case 1264: {
```
Comment (mfine) [inline c/src/rtcm3_sbp.c]: 1264? the haskell code has this as 1265 and 1266:

```
  (RTCM3Msg1265 m _rtcm3) -> SSR.gpsPhaseBiasConverter m
  (RTCM3Msg1266 m _rtcm3) -> SSR.glonassPhaseBiasConverter m
```

Comment (mfine) [reply c/src/rtcm3_sbp.c]: screen shot 2018-08-10 at 08 36 43

Comment (anth-cole) [reply c/src/rtcm3_sbp.c]: fixed

**Comment by mfine** [comment]
looks good!

**Comment by mfine** [comment]
Sample conversions in the ppp filter:

```
INPUT   (run_daemon)    rtcm-frame type: 1059
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    sbp-frame type: 1505
INPUT   (run_daemon)    rtcm-frame type: 1265
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
INPUT   (run_daemon)    sbp-frame type: 1510
```

**Pull request merged**